### DISCUSSION TITLE
Div gets inserted between layout and page components, cannot style it.

### DISCUSSION THREAD

**Discussion by VincentGillot** [vercel/next.js#41830] [Help]
Hello, I've been switching my portfolio from 12 to 13 and it appears there is a div randomly inserted between the layout and page component that has no ID nor class names. My home page needs a flex grow to occupy the whole width but since this div is not styleable and has no style it breaks the entire layout.
In 12 I had to add a css rule for div#__next to set it as flex: 1;, which i guess might be a similar situation except in 13 the div has no ID. Or maybe it comes from a different place. No idea how to fix this.

![image](https://user-images.githubusercontent.com/73160036/197964487-03218e9d-3705-44ec-a6b8-f85bd21c2705.png)

Layout:
![image](https://user-images.githubusercontent.com/73160036/197964576-0ec7a6d7-29d8-46b1-be0b-d99e9d345297.png)

Page:
![image](https://user-images.githubusercontent.com/73160036/197964988-ec90447c-03c4-4061-a6f7-38c87e7d3729.png)

**Comment by arnihermann** [comment]
I tried having a `template.tsx` file but to no avail and am also looking for some way to get rid of this intermediate node.
